### PR TITLE
Blog post for LWIMIQ June 26, 2017.

### PR DESCRIPTION
**Pull request by jvlcek**
@djberg96 and @JPrause  Please review

### CODE REVIEW

**Comment by djberg96** [comment]
👍 LGTM

**Comment by JPrause** [comment]
@imtayadeway @jntullo can you also have a look-see.

File: site/_posts/2017-06-26-migrating-migrations.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+
+## Deleted
+
+#### Related to the improvement noted above [Jason Frey](https://github.com/Fryguy) deleted the db/migrate directory.
+  * [Extract migrations to manageiq-schema repo](https://github.com/ManageIQ/manageiq/pull/15293)
+
+
+## Wrapping up
+
+That wraps it up for this week inManageIQ. To stay in touch and learn more about the project take a look at  [MangeIQ.ORG](http://manageiq.org/)
```
Comment (jntullo) [inline site/_posts/2017-06-26-migrating-migrations.md]: minor thing - needs space "in manageIQ"

**Comment by jntullo** [comment]
👍 LGTM

File: site/_posts/2017-06-26-migrating-migrations.md (hunk 1 above)
Comment (jvlcek) [inline site/_posts/2017-06-26-migrating-migrations.md]: @jntullo Good catch. I even missed that when I ran the source through spell checker!

**Comment by jvlcek** [comment]
@jntullo Thank you!

File: site/_posts/2017-06-26-migrating-migrations.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+
+## Deleted
+
+#### Related to the improvement noted above [Jason Frey](https://github.com/Fryguy) deleted the db/migrate directory.
+  * [Extract migrations to manageiq-schema repo](https://github.com/ManageIQ/manageiq/pull/15293)
+
+
+## Wrapping up
+
+That wraps it up for this week in ManageIQ. To stay in touch and learn more about the project take a look at  [MangeIQ.ORG](http://manageiq.org/)
```
Comment (cybette) [inline site/_posts/2017-06-26-migrating-migrations.md]: [MangeIQ.ORG] -> missing an 'a'

Comment (jvlcek) [inline site/_posts/2017-06-26-migrating-migrations.md]: Good Catch! @cybette Thank you!

**Comment by JPrause** [comment]
LGTM! Thanks @jvlcek

**Comment by jvlcek** [comment]
Thank you, @cybette, @jntullo and @JPrause!
I squashed down my typo-fix commits.

JoeV

File: site/_posts/2017-06-26-migrating-migrations.md [hunk 3]
```diff
@@ -0,0 +1,60 @@
+---
+title: "Last Week in ManageIQ: The DB migrations have migrated."
```
Comment (imtayadeway) [inline site/_posts/2017-06-26-migrating-migrations.md]: :heart:

**Comment by imtayadeway** [comment]
LGTM! Except that I would perhaps s/PR's/PRs/. Thanks, @jvlcek ! :bike:

**Comment by jvlcek** [comment]
@imtayadeway's Good points ! ;)

Thank you! JoeV

**Comment by jvlcek** [comment]
I squashed the typo/syntax commits

**Comment by syncrou** [comment]
@jvlcek - 👍 Looks great to me!  Thanks for the write up. I really like that the db repo split was under the deletions section!

**Pull request merged**